src/OpenTelemetry/Logs/ILogger/OpenTelemetryLoggingExtensions.cs

```diff
@@ -48,6 +51,9 @@ public static class OpenTelemetryLoggingExtensions
     /// </remarks>
     /// <param name="builder">The <see cref="ILoggingBuilder"/> to use.</param>
     /// <returns>The supplied <see cref="ILoggingBuilder"/> for call chaining.</returns>
+#if EXPOSE_EXPERIMENTAL_FEATURES
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
We could add a comment here explaining why we are adding this attribute.

Reply:
Updated